src/tbbmalloc_proxy/CMakeLists.txt

```diff
@@ -53,8 +53,8 @@ set_target_properties(tbbmalloc_proxy PROPERTIES
 if (UNIX AND NOT APPLE)
     # Avoid use of target_link_libraries here as it changes /DEF option to \DEF on Windows.
     set_target_properties(tbbmalloc_proxy PROPERTIES
-        LINK_FLAGS ${TBB_LINK_DEF_FILE_FLAG}${CMAKE_CURRENT_SOURCE_DIR}/def/${TBB_DEF_FILE_PREFIX}-proxy.def
-        LINK_DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/def/${TBB_DEF_FILE_PREFIX}-proxy.def
+        LINK_FLAGS ${TBB_LINK_DEF_FILE_FLAG} "\"${CMAKE_CURRENT_SOURCE_DIR}/def/${TBB_DEF_FILE_PREFIX}-proxy.def\""
+        LINK_DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/def/${TBB_DEF_FILE_PREFIX}-proxy.def"
```

Comment:
Should not it be removed as well?

Reply:
I decided not to touch `tbbmalloc_proxy`, since I don't really know how it functions with Linux LD_PRELOAD system, and there are only 2 def files anyway, so I don't think it's gonna be difficult to maintain.